### PR TITLE
Fix the number of tests for `forAll`

### PR DESCRIPTION
**Pull request by mohsen-ghaffari1992**

### CODE REVIEW

File: src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala [hunk 1]
```diff
@@ -6,6 +6,9 @@ import symsim.examples.concrete.mountaincar.MountainCar
 class ConcreteSarsaIsSarsaSpec
   extends SymSimSpec:
 
+  implicit override val generatorDrivenConfig =
```
Comment (wasowski) [inline src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala:9]: change `implicit` to `given`

Comment (wasowski) [reply src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala:9]: And we should move this to `SymSimSpec`

**Review by wasowski — CHANGES_REQUESTED** [review]

File: src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala (hunk 1 above)
Comment (mohsen-ghaffari1992) [reply src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala:9]: I moved it into SymSimSpec and it was fine but when I changed given to implicit, it started to complaining.
I am trying to solve the issue but I thought maybe there is some different syntax that you already know.

File: src/test/scala/symsim/examples/concrete/simplebandit/BanditIsAgentSpec.scala [hunk 2]
```diff
@@ -6,10 +6,15 @@ import laws.EpisodicLaws
...
+
    checkAll ("concrete.simplebandit.Bandit is an Agent", AgentLaws (BanditObjConst).laws)
    checkAll ("concrete.simplebandit.Bandit is Episodic", EpisodicLaws (BanditObjConst).laws)
    
 class BanditIsAgentSpecGaussian
    extends SymSimSpec:
+
+   implicit override val generatorDrivenConfig =
+      PropertyCheckConfiguration(minSuccessful = 100)
+
```
Comment (wasowski) [inline src/test/scala/symsim/examples/concrete/simplebandit/BanditIsAgentSpec.scala:18]: This should be removed, shouldn't it? It is already included via SymSimSpec?

**Review by wasowski — CHANGES_REQUESTED** [review]

**Review by wasowski — APPROVED** [review]

**Pull request merged**